CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+## v1.0.0
+
+- Remove lsp4j, as per https://github.com/clojure-lsp/lsp4clj/issues/8
+This is essentially a rewrite of lsp4clj. Users of lsp4clj v0.4.3 and earlier
+are encouraged to upgrade. Bug fixes for these earlier versions will be
+considered, but the lsp4j-based version of this library will not receive
+long-term support. For an example of how to use the new version of the library,
+see https://github.com/clojure-lsp/clojure-lsp/pull/1117
```

Comment:
Since the current version is alpha, I'm totally fine merging v1 into master soon after it's running fine on clojure-lsp

Reply:
@ericdallo how do you want to test v1 with clojure-lsp? I think it's easiest to pin it to a `:git/sha` near the tip of v1, but we could release v1 to clojars since the project name is changing.

Reply:
I'm ok with using git/sha until we think it's stable enough and with no bugs